docs/templates/tutorials/distributed-tuning.md

```diff
@@ -0,0 +1,122 @@
+# Distributed Tuning
+
+Keras Tuner makes it easy to perform distributed hyperparameter search. No changes to your code are needed to scale up from running single-threaded locally to running on dozens or hundreds of workers in parallel. Distributed Keras Tuner uses a chief-worker model. The chief runs a service to which the workers report results and query for the hyperparameters to try next. The chief should be run on a single-CPU worker (or alternatively as a separate process on one of the workers).
+
+### Configuring distributed mode
+
+Configuring distributed mode for Keras Tuner only requires setting three environment variables:
+
+**KERASTUNER_TUNER_ID**: This should be set to "chief" for the chief process. Other workers should be passed a unique ID (by convention, "tuner0", "tuner1", etc).
+
+**KERASTUNER_ORACLE_IP**: The IP address of the chief service.
+
+**KERASTUNER_ORACLE_PORT**: The port that the chief service should run on.
+
+The same code can be run on all workers. Additional considerations for distributed mode are:
+
+- All workers should have access to a centralized file system to which they can write their results.
+- All workers should be able to access the necessary training and validation data needed for tuning.
+- To support fault-tolerance, `overwrite` should be kept as `False` in `Tuner.__init__` (`False` is the default).
+
+Example chief setup (sample code for `run_tuning.py` at bottom of page):
+
+```bash
+export KERASTUNER_TUNER_ID="chief"
+export KERASTUNER_ORACLE_IP="127.0.0.1"
+export KERASTUNER_ORACLE_PORT="8000"
+python run_tuning.py
+```
+
+Example worker setup:
+
+```bash
+export KERASTUNER_TUNER_ID="tuner0"
+export KERASTUNER_ORACLE_IP="127.0.0.1"
+export KERASTUNER_ORACLE_PORT="8000"
+python run_tuning.py
+```
+
+### Data parallelism with tf.distribute
+
+Keras Tuner also supports data parallelism via `tf.distribute`. Data parallelism and distributed tuning can be combined. For example, if you have 10 workers with 4 GPUs on each worker, you can run 10 parallel trials with each trial training on 4 GPUs by using `tf.distribute.MirroredStrategy`. You can also run each trial on TPUs via `tf.distribute.experimental.TPUStrategy`. Currently `tf.distribute.MultiWorkerMirroredStrategy` is not supported, but support for this is on the roadmap.
```

Comment:
tf.distribute: add link pointing to DistStrat guide on tf.org. Likewise for mentions of specific strategies: link to the TF docs

Reply:
Done